code/obj/item/uplinks.dm

```diff
@@ -1275,6 +1289,13 @@ Note: Add new traitor items to syndicate_buylist.dm, not here.
 								new B.item3(get_turf(src))
 
 							B.run_on_spawn(A, usr, FALSE, src)
+
+							// Remember purchased item for the crew credits
+							var/datum/antagonist/nuclear_operative/antagonist_role = usr.mind?.get_antagonist(ROLE_NUKEOP)
+							if (antagonist_role == null )
+								antagonist_role = usr.mind?.get_antagonist(ROLE_NUKEOP_COMMANDER)
+							antagonist_role.uplink_items.Add(B)
```

Comment:
```suggestion
							antagonist_role?.uplink_items.Add(B)
```
Can we asset it isn't null?

Reply:
I changed instances of this as well as the original code it was copied from (just above these lines)

Reply:
The `istype()` checks were sufficient for previous instances of this.

Reply:
Ah, right. I definitely got this area mixed up with the others. Thank you! It's back to original now